docs/sql-data-sources-jdbc.md

```diff
@@ -1335,3 +1335,109 @@ as the activated JDBC Driver.
     </tr>
   </tbody>
 </table>
+
+### Mapping Spark SQL Data Types to Oracle
+
+The below table describes the data type conversions from Spark SQL Data Types to Oracle data types,
+when creating, altering, or writing data to an Oracle table using the built-in jdbc data source with
+the Oracle JDBC as the activated JDBC Driver.
+
+<table>
+  <thead>
+    <tr>
+      <th><b>Spark SQL Data Type</b></th>
+      <th><b>Oracle Data Type</b></th>
+      <th><b>Remarks</b></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>BooleanType</td>
+      <td>NUMBER(1, 0)</td>
+      <td>BooleanType maps to NUMBER(1, 0) as BOOLEAN is introduced since Oracle Release 23c</td>
```

Comment:
we might introduce a configuration for users to map BOOLEAN instead.

Reply:
Is it because `Oracle Database 23c` is introduced on April 2023?

Reply:
Yes, previous versions didn't have BOOLEAN